### PR TITLE
Fix #9 Added another python example. 

### LINKED ISSUE #9: Request to community: Please submit examples of Apps :rocket:
We would like to have lots of examples in this repo, written in any language of you're choosing! If you have an example to share, please open a pull request in this repo with your example App! :rocket:

### PR DESCRIPTION
**Pull request by carydrew**
#### Summary

Multiple Changes.

1. Created a new example using a Dynamic Select field type that also used refresh: true to add in a second field when a user selects an option. 
2. I added in a very basic models class that is imported into this. This is formatted off the go [documentation](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps). This allowed the file to be typed in accordance with [PEP 484](https://peps.python.org/pep-0484/) which makes it easier to correctly modify items. 
3. Removed the line `'app_type': 'http',` from the manifest function as that is not needed according to the [documentation](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Manifest). The file was also formatted on save so it looks like more changes. Those are just length, removing unneeded f string and single quotes to double quotes. The formatting was done by Ruff/Black to bring better in line with[ PEP 8](https://peps.python.org/pep-0008/).

#### Ticket Link

#9

### CODE REVIEW

File: python/dynamic-hello-world/src/static/icon.png
Comment (azigler) [inline python/dynamic-hello-world/src/static/icon.png:1]: Could we just reference the pre-existing file instead of copying it into another directory?

**Review by azigler — APPROVED** [review]
Looks great to me -- thanks for adding a new example! 👍 🤩  Tagging @mickmister to take a look when he has a chance, for anything else that might stand out.

There's an opportunity to explain this example in the [documentation](https://developers.mattermost.com/integrate/apps/quickstart/) -- if this is merged I recommend opening an issue in https://github.com/mattermost/mattermost-developer-documentation so we can track and add that.

File: python/dynamic-hello-world/src/static/icon.png
Comment (carydrew) [reply python/dynamic-hello-world/src/static/icon.png:1]: Yes we can do that. I copied it over in that way to keep the examples a bit more separate, but I can point to the file in the hello-world directory if you want.

File: python/mattermost_models/models.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        label: str
+        value: str
+        icon_data: str
+
+    class Lookup(TypedDict, total=False):
+        """Lookup field for the forms"""
+
+        # Label is the display name/label for the option's value.
+        type: Literal["ok", "error", "form", "navigate"]
+        data: List[Models.SelectField]
```
Comment (mickmister) [inline python/mattermost_models/models.py:49]: Amazing, bringing this into classes like this :+1:

**Review by mickmister — APPROVED** [review]
LGTM, thanks @carydrew!

File: python/dynamic-hello-world/src/static/icon.png
Comment (carydrew) [reply python/dynamic-hello-world/src/static/icon.png:1]: Honestly I am indifferent as well. If referencing the pre-existing file it should be easy enough to make it obvious where it is pulling from to not cause any confusion.

**Pull request merged**